build/templates/package-validation.yml

```diff
@@ -47,7 +48,7 @@ steps:
       $ErrorActionPreference = 'Stop'
 
       # Create app with defaults
-      dotnet new unoapp -o UnoApp1 ${{ parameters.arguments }}
+      dotnet new unoapp -skip -o UnoApp1 ${{ parameters.arguments }}
```

Comment:
What does the `-skip` flag do?

Reply:
Avoids doing a restore as part of creating template - this makes the whole process quicker since the dotnet build does a restore anyhow